### PR TITLE
Navbar accessibility

### PR DESCRIPTION
**Pull request by sabakhilji**
Closes issue #441

### CODE REVIEW

**Review by shaavan — COMMENTED** [review]
I agree with the approach that you have taken to making the app more accessible.
Way to go, @sabakhilji! 🚀

I have a small comment to add to make this PR even better.

File: i18n/locales/en.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ const translations = {
     coming_soon: 'Coming soon',
     start_chapter: 'Start chapter',
     back: 'Back',
+    poweroff: 'back to chapter selection page',
```
Comment (shaavan) [inline i18n/locales/en.ts:14]: Small nit:

Capitalizing the sentence will look better :)

Also, I think we can remove "page" from the phrase to make it concise.

```suggestion
    poweroff: 'Back to chapter selection',
```

File: components/Navbar/HelpLink.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export default function HelpLink({ params }: { params: any }) {
           target="_blank"
           href="https://forms.gle/NVcg9ukPvUBYjw1u7"
         >
-          ?
```
Comment (GBKS) [inline components/Navbar/HelpLink.tsx:39]: By the way, the "?" should be Nunito and not Caveat.

Comment (sabakhilji) [reply components/Navbar/HelpLink.tsx:39]: Hey, @GBKS I am not sure about that  '?'. The only line code which I added is related to a screen reader. 
Also, I would like to ask about the title of the power-off button. Right now I changed it to 'back to chapter selection page'. Please let me know what's your opinion about it.

Comment (GBKS) [reply components/Navbar/HelpLink.tsx:36]: Yep, Ben is correct. The new title also looks good.

**Pull request merged**